tests/robustness/testdata/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Can we just ignore the `tests/robustness/testdata` directory in the top level `.gitignore`?